AMaDda/Model/UserDefaultsStateManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FamilyContactManager.swift
+//  AMaDda
+//
+//  Created by Lee Myeonghwan on 2022/07/20.
+//
+
+import Foundation
+
+final class UserDefaultsStateManager {
+    private var today: Date {
+        return convertKoreaDate(Date())
+    }
+    func userEnteredApp() {
+        guard var finalEnteredDate = UserDefaults.standard.finalEnteredDate else {
+            UserDefaults.standard.finalEnteredDate = today
+            return
+        }
+        finalEnteredDate = convertKoreaDate(finalEnteredDate)
+        let offsetDateComponents = Calendar.current.dateComponents([.day], from: finalEnteredDate, to: today)
+        guard let offsetDay = offsetDateComponents.day else { return }
+        if offsetDay == 0 {
+            return
```

Comment:
```suggestion
        guard let offsetDay = offsetDateComponents.day, offsetDay != 0 else { return }
```

p3;

guard 구문과 if 구문을 하나로 합쳐서 표현하는 게 `offsetDay 에 대한 조건이 해당하지 않을 시 아래 구문을 실행하지 않음` 의 의미를 명확히 전달할 수 있을 거 같은데 어떻게 생각하실까요??

Reply:
그렇네요!! 확인하고 수정했습니다!